pyslackers_website/slack/views.py

```diff
@@ -20,8 +21,10 @@ class SlackInvite(FormView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        slack_member_tz_count = dumps(cache.get('slack_member_tz_count', []))
```

Comment:
I believe that the controller/view should be dumb and not know how this data intends to be used (I think that is a display concern strictly). I think it would be best to make a new [template tag](https://docs.djangoproject.com/en/1.11/ref/templates/builtins/) that converts a python object to JSON for the view.

Along those lines, new additions require tests - if you can add something about context I'd like to see tests that operate like the `blog/tests.py` for just this new context item (I can back fill the other context I already have here).

Reply:
Cool, I was hesitant to add a new template tag for this but it's doable.

I'll check out the tests as well.